tests/test_context.py

```diff
@@ -19,7 +20,7 @@
 
 """This pytest module tries to test the correctness of the class Context."""
 
-from palace import current_context, Context, MessageHandler
+from palace import current_context, Context, DistanceModel, MessageHandler
```

Comment:
Due to the recent change of the design by Phong, DistanceModel enum is no longer there. Please read source code for more information.

Reply:
I noticed you haven't updated this yet. Please pull from master first.